app/assets/javascripts/discourse/app/instance-initializers/auth-complete.js

```diff
@@ -53,7 +53,11 @@ export default {
                 ...properties,
               };
 
-              router.trigger("showLogin", props);
+              router.transitionTo("login").then((login) => {
+                for (const [key, value] of Object.entries(props)) {
+                  login.controller.set(key, value);
+                }
+              });
```

Comment:
Reverting this to the previous format, not quite sure why `router.trigger` didn't work here.